Makefile

```diff
@@ -160,3 +161,7 @@ post-release: tools/yq
 .PHONY: generate
 generate: tools/stringer
 	go generate ./...
+
+.bin/ory: Makefile
+	curl https://raw.githubusercontent.com/ory/meta/master/install.sh | bash -s -- -b .bin ory v0.1.43
```

Comment:
How do we update the version here? Manually? All other dependencies are tracked in Homebrew or in `.bin/gobin/go.mod`. Can we put it there?

Reply:
We already have the Ory CLI checked in through homebrew, this can just be skipped all together :wink:

Reply:
Thanks for the feedback! This brings up a larger conversation (than fits into this PR), so I have opened https://github.com/ory-corp/general/issues/754 to discuss the various approaches to install dependencies, the thinking behind them, and how we might simplify the current complexity around dependency management in all repos. 

@hperl yes I was planning on bumping the version manually if needed, through a PR that verifies that everything still works. That's the most simple, straightforward, and robust solution in my experience, and anybody can do it. I created https://github.com/kevgo/mrt to automatically perform such updates for all Ory repos if needed.

@zepatrik how do I install the Ory CLI using Homebrew on Linux (Debian Bullseye)? I just ran `make sdk` and `make tools/cli` (which I guess will create a `tools/ory` command?) and both ran for 5 minutes each and then errored with:

```
Warning: No available formula with the name "keto/tools/go-swagger@0.30.0".
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
make: *** [Makefile:37: tools/go-swagger] Error 1
```

Given that the ORY CLI is a simple stand-alone Go binary with a custom binary name, why aren't we installing it through its cross-platform installer script? That takes only a few seconds.

Reply:
> Given that the ORY CLI is a simple stand-alone Go binary with a custom binary name, why aren't we installing it through its cross-platform installer script? That takes only a few seconds.

This will work for Ory CLI, but not for `protoc` :disappointed: 
I am not sure why it failed for you, but I agree that the brew variant sucks. I will write an install script for protoc instead that pulls the binary from github. I'll try to contribute it to https://github.com/protocolbuffers/protobuf so we don't have to maintain it.

Reply:
Sounds great! Here is a super simple template in case you need one: https://github.com/git-town/git-town/blob/main/website/scripts/install_mdbook.